#!/usr/bin/env python3

PACKAGE = "task0_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("color",    str_t,    0, "color selected",  "red")

gen.add("h_min",    int_t,    0, "hue min value", 0,  0, 255)
gen.add("h_max",    int_t,    0, "hue max value", 255,  0, 255)
gen.add("s_min",    int_t,    0, "sat min value", 0,  0, 255)
gen.add("s_max",    int_t,    0, "sat max value", 255,  0, 255)
gen.add("v_min",    int_t,    0, "val min value", 0,  0, 255)
gen.add("v_max",    int_t,    0, "val max value", 255,  0, 255)

exit(gen.generate(PACKAGE, "dynamic_hsv", "HSV"))
